build.gradle

```diff
@@ -79,3 +79,36 @@ processResources {
 jar {
     from configurations.shade.collect { it.isDirectory() ? it : zipTree(it) }
 }
+
+// utility tasks
+task deploy {
+	group = 'build'
+	description = 'Generates a jar and a source jar for release'
+	
+	dependsOn jar
```

Comment:
TBH I'm not sure why we need a separate task for `deploy` since you can just run `build` to get the release and source jars (which now contain LuaJ and such)